firefox-ios/Client/Frontend/Browser/Tabs/Views/TabDisplayView.swift

```diff
@@ -292,6 +292,41 @@ class TabDisplayView: UIView,
     }
 }
 
+// MARK: - SwipeAnimatorDelegate
+extension TabDisplayView: SwipeAnimatorDelegate {
```

Comment:
nit: we prefer adding delegate conformance in the once place and avoiding extensions as a means of breaking up a class.
https://github.com/mozilla-mobile/firefox-ios/wiki/When-To-Use-Extensions

Reply:
Done! Also removed an unused variable in the protocal parameters for `func swipeAnimator` in the protocol.